test/src/e2e.dynval/dv.n'.test.ts

```diff
@@ -136,4 +136,269 @@ describe("Dynamic Validator N -> N'", function() {
             );
         });
     });
+
+    describe("2. Jail one of the validator + increase the deposit of a candidate who doesn’t have enough deposit", async function() {
+        // alice : Elected as a validator, but does not send precommits and does not propose.
+        //   Alice should be jailed.
+        // betty : Not elected as validator because of small deposit. She deposits more CCC in the first term.
+        //   betty should be a validator in the second term.
+        const allDynNodes = withNodes(this, {
+            promiseExpect,
+            overrideParams: {
+                termSeconds: TERM_SECONDS
+            },
+            validators: [
+                { signer: alice, delegation: 5000, deposit: 100000 },
+                { signer: betty, delegation: 5000, deposit: 100 },
+                ...otherDynValidators.map((validator, index) => ({
+                    signer: validator,
+                    delegation: 5000 - index,
+                    deposit: 100000
+                }))
+            ],
+            onBeforeEnable: async allDynNodes => {
+                const aliceNode = allDynNodes[0];
+                // Kill the alice node first to make alice not to participate in the term 1.
+                await aliceNode.clean();
+
+                return allDynNodes.slice(1);
+            }
+        });
+
+        it("Alice should get out of the committee and Betty should be included in the committee", async function() {
+            const margin = 1.1;
+            const [bettyNode, ...otherDynNodes] = allDynNodes;
```

Comment:
It is now
```
const [_aliceNode, bettyNode, ...otherDynNodes] = allDynNodes;
```
with this changes https://github.com/CodeChain-io/codechain/pull/1688